<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exportSelectorDialog</class>
 <widget class="QDialog" name="exportSelectorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>189</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>190</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Select exports</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QCheckBox" name="expSoundsCfgEnable">
     <property name="text">
      <string>Export sounds configuration</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="expImagesCfgEnable">
     <property name="text">
      <string>Export images configuration</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="expAPNCfgEnable">
     <property name="text">
      <string>Export APN settings</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="expDeviceCfgEnable">
     <property name="text">
      <string>Export device configuration</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="expContactNumsCfgEnable">
     <property name="text">
      <string>Export contact numbers</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="expSMSCfgEnable">
     <property name="text">
      <string>Export SMS contents</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="acceptButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>expDeviceCfgEnable</tabstop>
  <tabstop>expImagesCfgEnable</tabstop>
  <tabstop>expSoundsCfgEnable</tabstop>
  <tabstop>expContactNumsCfgEnable</tabstop>
  <tabstop>expSMSCfgEnable</tabstop>
  <tabstop>expAPNCfgEnable</tabstop>
  <tabstop>acceptButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
